docs/imviz/plugins.rst

```diff
@@ -74,6 +74,28 @@ parameters, shape, and orientation (if applicable) all update concurrently.
 
 Angle is counter-clockwise rotation around the center in degrees.
 
+.. _markers-plugin:
+
+Markers
+=======
+
+This plugin allows for interactively creating markers in any viewer and logging information about
+the location of that marker along with the applicable data and viewer labels into a table.
+
+With the plugin open in the tray, mouse over any viewer and press the "m" key to log the information
+displayed in the app toolbar into the table.  The markers remain at that fixed pixel-position in
+the viewer they were created (regardless of changes to the underlying data or linking) and are only
+visible when the plugin is opened.
```

Comment:
Now that markers actually follow linking, we can remove some text here?

```suggestion
displayed in the app toolbar into the table.  The markers remain in
the viewer they were created and are only
visible when the plugin is opened.
```

Reply:
Is my suggestion here irrelevant? That's ok if it is. I just want sure this comment did not fall through the cracks.

Reply:
I think the original text is still necessary, markers _record_ the image on which they were created/logged, but ultimately will always show at the _axes coordinates_ in which they were created.